index.html

```diff
@@ -80,25 +80,25 @@ <h1 class="title">Learn UX Design</h1>
     <!-- INFO-start -->
     <div class="container">
         <div class="row infocont">
-       
-                <div class="herinfo col-12 col-md-6 col-lg-3 col-xl-3">
-                    <h3>2125 </h3>
-                    Enrolled Students
-                </div>
-                <div class="herinfo col-12 col-md-6 col-lg-3 col-xl-3">
-                    <h3>5 Weeks</h3>
-                    Estimated Complete Time
-                </div>
-                <div class="herinfo col-12 col-md-6 col-lg-3 col-xl-3">
-                    <h3>25 September</h3>
-                    Estimated Complete Time
-                </div>
-                <div class="herinfo col-12 col-md-6 col-lg-3 col-xl-3">
-                    <h3>English</h3>
-                    Estimated Complete Time
-                </div>
+
+            <div class="herinfo col-12 col-md-6 col-lg-3 col-xl-3">
+                <h3>2125 </h3>
+                Enrolled Students
```

Comment:
Šis tekstas taip pat būtų gerai, jog būtų kažkokiame tag'e.
Dėl galimybės keisti stilių ir suprasti hierarchiją.